person.rb

```diff
@@ -1,9 +1,11 @@
+require './corrector'
 # Create class person
-class Person
+class Person < Corrector
```

Comment:
- [ ] Person should not be based on the corrector class, there should be an instance of the corrector class set in the person class on its initialization like this:

```ruby
class Person
  .
  .

  def initialize(age, name = "Unknown", parent_permission = true)
    .
    @corrector = Corrector.new()
    .
    .
  end

  .
  .
end
```

Please make this small change to comply with the project requirements.

[Support article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes)

Reply:
Thanks for the feedback! to be honest i was a bit confused about that part!